cmd/shutdown_test.go

```diff
@@ -0,0 +1,44 @@
+package main
+
+import (
+	"io"
+	"testing"
+	"time"
+)
+
+type CloserMock struct {
+	name     string
+	expected func() error
+}
+
+func NewCloserMock(name string, expected func() error) io.Closer {
+	return CloserMock{
+		name:     name,
+		expected: expected,
+	}
+}
+
+func (cm CloserMock) Close() error {
+	return cm.expected()
+}
+
+func TestGracefulShutdownSuccess(t *testing.T) {
+	timeout := time.Second * 10
```

Comment:
You can use your constant shutdownTimeout there. It is the same value. It is unnecessary create new variable

Reply:
fixed